spot-client/src/common/utils/meeting.js

```diff
@@ -129,3 +145,34 @@ export function parseMeetingUrl(url) {
         path: pathParts.join('/')
     };
 }
+
+/**
+ * Builds the video constraints based on the preferred aspect ratio and resolution.
+ *
+ * @param {string} preferredAspectRatio - The aspect ratio of choice.
+ * @param {string} preferredResolution - The resolution of choice.
+ * @returns {boolean}
+ */
+export const getVideoConstraints = (preferredAspectRatio, preferredResolution) => {
```

Comment:
Where does the aspect ratio come from? I think we should use 16 / 9 at all times. Regardless of the size of the app window, cameras support stabndard aspect ratios, and requiring 16 / 9 for an in-room device sounds very reasonable.

Reply:
I used [this](https://github.com/jitsi/jitsi-meet-spot/blob/master/spot-client/src/common/app-state/config/selectors.js#L276), but I'll switch it to 16 / 9.